src/index.ts

```diff
@@ -1,15 +1,32 @@
 console.clear();
 
-let personas = [12, 36, 18, 40, 8, 32, 46, 22, 25, 39, 18, 19];
+let personas: number[] = new Array(270);
+let edadMinima: number = 18;
+let edadMaxima: number = 40;
 
-function esMayorQue21(numero) {
-  if (numero > 21) {
-    console.log(numero + " es mayor que 21.");
+let cantidadMenores21: number = 0;
+let cantidadMayores21: number = 0;
+
+for (let indice = 0; indice < 270; indice++) {
+  let numeroAleatorio = Math.floor(
+    Math.random() * (edadMaxima - edadMinima) + edadMinima
+  );
+  personas[indice] = numeroAleatorio;
+}
+
+for (let indice = 0; indice < 270; indice++) {
+  let edad = personas[indice];
+  if (edad > 21) {
+    cantidadMayores21 = cantidadMayores21 + 1;
   } else {
-    console.log(numero + " no es mayor que 21.");
+    cantidadMenores21 = cantidadMenores21 + 1;
   }
 }
```

Comment:
Aca lo que hacemos es un nuevo **for**:
- Este nuevo for lo que hace es recorrer todo el arreglo **personas**, que ya tiene los valores aleatorios cargados en el for anterior.
- En este caso volvemos a recorrer el arreglo pero para contar la cantidad de personas menores a 21 y mayores a 21.
- Entonces dentro del for lo que hacemos es agregar un if, que consulta si la edad es mayor a 21, suma 1 a la variable cantidadMayores21, y si es menor a 21 entonces suma 1 a la variable cantidadMenores21.